Updating NLS_DATE_FORMAT config for Analytics

### Current Limitation

NLS_DATE_FORMAT should be configured as 'RRRR/fmMM/fmDD' since some Oracle versions result in errors (refer to below) without this.  

`ERROR {io.siddhi.core.aggregation.persistedaggregation.CudStreamProcessorQueueManager} - Error occurred while executing the aggregation for data between 1712534400000 - 1712620800000 for duration DAYS. Attempted re-executing the query for 9 seconds. This Should be investigated since this will lead to a data mismatch
io.siddhi.core.exception.SiddhiAppRuntimeException: Error in manipulating records from datasource 'APIM_ANALYTICS_DB': ORA-01830: date format picture ends before converting entire input string

```
https://docs.oracle.com/error-help/db/ora-01830/
	at io.siddhi.extension.execution.rdbms.CUDStreamProcessor.process(CUDStreamProcessor.java:248)
	at io.siddhi.core.query.processor.stream.StreamProcessor.processEventChunk(StreamProcessor.java:41)
	at io.siddhi.core.query.processor.stream.AbstractStreamProcessor.process(AbstractStreamProcessor.java:132)
	at io.siddhi.core.aggregation.persistedaggregation.CudStreamProcessorQueueManager.run(CudStreamProcessorQueueManager.java:66)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.sql.BatchUpdateException: ORA-01830: date format picture ends before converting entire input string
```

Refer internal: https://github.com/wso2-enterprise/wso2-apim-internal/issues/5980


### Suggested Improvement

Better to update [1] with this information.
[1] https://apim.docs.wso2.com/en/4.0.0/use-cases/streaming-usecase/summarizing-data/


### Version

3.2.0

--- a/en/docs/use-cases/streaming-usecase/summarizing-data.md
+++ b/en/docs/use-cases/streaming-usecase/summarizing-data.md
@@ -78,8 +78,14 @@ Observe the following in the above Siddhi application:
     Also when using persisted aggregation with Oracle, add below configuration in the datasource configuration,
 
     ```
-    connectionInitSql: alter session set NLS_DATE_FORMAT='RRRR/fmMM/fmDD'
-       
+    connectionInitSql: alter session set NLS_DATE_FORMAT='YYYY-MM-DD HH24:MI:SS'
+
+    ```
+    !!! note
+    If you encounter the error `ORA-01830: date format picture ends before converting entire input string`, use the following configuration instead:
+    `connectionInitSql: alter session set NLS_DATE_FORMAT='RRRR/fmMM/fmDD'`
+
+    ```
     eg:
        
      - name: APIM_ANALYTICS_DB
@@ -96,9 +102,10 @@ Observe the following in the above Siddhi application:
              maxPoolSize: 50
              idleTimeout: 60000
              connectionTestQuery: SELECT 1 FROM DUAL
-             connectionInitSql: alter session set NLS_DATE_FORMAT='RRRR/fmMM/fmDD'
+             connectionInitSql: alter session set NLS_DATE_FORMAT='YYYY-MM-DD HH24:MI:SS'
              validationTimeout: 30000
              isAutoCommit: false
+    ```
                
     ```
     For an example please refer to the following query which will be executed on the database to update the table for below sample Aggregation ,

